xgi/algorithms/centrality.py

```diff
@@ -112,8 +112,8 @@ def h_eigenvector_centrality(H, max_iter=100, tol=1e-6):
 
     new_H = convert_labels_to_integers(H, "old-label")
 
-    f = lambda v, m: np.power(v, 1.0 / m)
-    g = lambda v, x: np.prod(v[list(x)])
+    f = lambda v, m: np.power(v, 1.0 / m)  # noqa: E731
```

Comment:
the comment tells ruff to ignore a potential error?

Reply:
Yeah. For reference, you can see the explanation of each rule using `ruff rule <rule_number>`:

```bash
$ ruff rule E731
```
# lambda-assignment (E731)

Derived from the **pycodestyle** linter.

Autofix is sometimes available.

## What it does
Checks for lambda expressions which are assigned to a variable.

## Why is this bad?
Per PEP 8, you should "Always use a def statement instead of an assignment
statement that binds a lambda expression directly to an identifier."

Using a `def` statement leads to better tracebacks, and the assignment
itself negates the primary benefit of using a `lambda` expression (i.e.,
that it can be embedded inside another expression).

## Example
```python
f = lambda x: 2*x
```

Use instead:
```python
def f(x):
   return 2 * x
```

## References
- [PEP 8](https://peps.python.org/pep-0008/#programming-recommendations)

Reply:
I usually try to abide by this rule but in this case it makes sense to assign to a lambda, I think.